setup.py

```diff
@@ -21,7 +22,7 @@
     ],
     entry_points={
         'console_scripts': [
-            'webapp = app',
+            'govstat = app',
```

Comment:
What exactly does this entail? Should this allow someone to launch the webserver with a CLI command `govstat`? That's not working for me.

Reply:
Yep it should. Did you perform `pip install -e .`? Or rerun `pip install .`?

Reply:
While I can "launch" the command `govstat` anywhere, `gunicorn` won't detect it, but that is because my system level `gunicorn` is being detected and ran. I wonder if its because of python3?

Reply:
![image](https://user-images.githubusercontent.com/5642476/154324848-6cdccad0-f3be-4613-a5f2-f95b599b29e5.png)

I tried `pip install -e .`. Do you get this error on your machine?

Reply:
disc?

Reply:
after a rebase i can see the same error, we can remove the entrypoint for now in setup.py and tackle it in another PR if we want to bring the feature back.